linux_setns_init.go

```diff
@@ -0,0 +1,33 @@
+// +build linux
```

Comment:
Would it be possible to get all inits in the same package? If not, no big deal :) I know there are dependency issues.